dashboard/test/test_helper.rb

```diff
@@ -170,6 +170,19 @@ def assert_same_keys(a, b, a_name = 'A', b_name = 'B', message = "Keys don't mat
   Found in #{a_name} but not #{b_name}: #{(a.keys - b.keys).join(', ')}
   Found in #{b_name} but not #{a_name}: #{(b.keys - a.keys).join(', ')})
   end
+
+  # Given a regular expression and a block, ensure that the block raises an
+  # exception with a message matching the regular expression.
+  def assert_raises_matching(matcher)
```

Comment:
Inspired by [the Chai assert.throws method](http://chaijs.com/api/assert/#method_throws) which lets you check the error message thrown.  Useful in this case since we just `raise "some message"` and want to make sure we hit the correct error case in our test.